inc/Engine/Optimization/RUCSS/Database/Queries/ResourcesQuery.php

```diff
@@ -125,4 +125,24 @@ public function create_or_update( array $resource ) {
 		return $db_row->id;
 	}
 
+	/**
+	 * Remove a resource from the table (if it is there).
+	 *
+	 * @since 3.9
+	 *
+	 * @param string $url URL of the item to remove.
+	 *
+	 * @return bool|int ID of the resource row removed, or false if was not in the table.
+	 */
+	public function remove( $url ) {
```

Comment:
**small enhancement:**

make this method more descriptive like renaming it to be `remove_by_url`.

what do u think ?

Reply:
Agreed